app/Providers/FastlyProvider.php

```diff
@@ -64,8 +64,87 @@ public function boot()
         });
 
         //deleted
+         Redirect::deleted(function($redirect) {
+            Log::info(sprintf('Deleting Fastly redirect: %s => %s (%d)',
+                $redirect->path, $redirect->target, $redirect->http_status));
+
+
+            // Here's the hacky part....
+            $fastly_key = '30342701a122f58e935c9fe1024b8bc8';
```

Comment:
Let's definitely put these into environment variables in `.env` so they're not in the repo, and then grab them from a config array in [services.php](https://github.com/DoSomething/nightwing/blob/master/config/services.php).